config/ucx.conf

```diff
@@ -12,3 +13,15 @@ UCX_IB_SEND_OVERHEAD=bcopy:5ns,cqe:50ns,db:400ns,wqe_fetch:350ns,wqe_post:100ns
 UCX_MM_SEND_OVERHEAD=am_short:40ns,am_bcopy:220ns
 UCX_MM_RECV_OVERHEAD=am_short:40ns,am_bcopy:220ns
 UCX_RCACHE_OVERHEAD=360ns
+
+[AMD Rome]
+CPU model=Rome
+UCX_DISTANCE_BW=phb:auto,node:17000MBs,sys:5100MBs
+
+[AMD Milan]
+CPU model=Milan
+UCX_DISTANCE_BW=phb:auto,node:17000MBs,sys:5100MBs
+
+[AMD Genoa]
+CPU model=Genoa
+UCX_DISTANCE_BW=phb:auto,node:17000MBs,sys:5100MBs
```

Comment:
1. Do we need something for Intel / ARM?
2. Are these settings based on DGX servers? since it depends if there is PCIe switch on the platform, not only on the system CPU

Reply:
1. Probably. Please note that I have not introduced any new values in this PR. The values are moved to the configuration file from topo.c file.
2. The values were introduced by the following PRs: https://github.com/openucx/ucx/pull/7390 and https://github.com/openucx/ucx/pull/9389.

Reply:
ok